### DISCUSSION TITLE
loco review on youtube

### DISCUSSION THREAD

**Discussion by lerina** [loco-rs/loco#802] [General]
wow [Code to the Moon](https://www.youtube.com/@codetothemoon) made a video about Loco_rs    <https://www.youtube.com/watch?v=7utPutDORb4>

Could be a good time to scrap the web and mention people that made articles, projects or guides relevant to Loco :-)

**Comment by jondot** [comment]
Thanks, yes, a new page here: https://github.com/loco-rs/loco/blob/master/docs-site/content/docs/resources/around-the-web.md
